### PR TITLE
maintainers: mobile: add members and packages

### PR DESCRIPTION
**Pull request by tomfitzhenry**
###### Description of changes

To distribute nixpkgs PR review load relating to Linux mobile packages, let's widen the net on contributors that are interested in maintaining Linux mobile packages.

This PR:
* adds members to the `mobile` team: myself, @Mindavi (as discussed in Matrix), @Pacman99 (see below comment), @zhaofengli (see below comment)
* adds 'mobile' team to the maintainers of some Linux mobile packages that I already maintain. I think there are other candidates too, e.g. Phosh.

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Pacman99** [comment]
I like the plan and I would be happy to join. I haven't used mobile linux in a bit, but I am still interested in it and plan to eventually. I still stay somewhat updated about the mobile linux stack.

**Comment by zhaofengli** [comment]
Same for me. My PinePhone has collected a bit of dust but I do want to come back at some point.

**Review by Mindavi — APPROVED** [review]
Hopefully this helps giving quicker notifications and getting faster reviews, I like the idea

File: maintainers/team-list.nix [hunk 1]
```diff
@@ -467,7 +467,9 @@ with lib.maintainers; {
 
   mobile = {
     members = [
+      mindavi
       samueldr
+      tomfitzhenry
```
Comment (samueldr) [inline maintainers/team-list.nix:472]: What name could it have, such that there is a distinction between the *Mobile NixOS* maintainers, and the *Mobile ecosystems in Nixpkgs* maintainers?

TBF `mobile` hasn't been used yet, but if/when generic things are upstreamed back into Nixpkgs, they would be maintained by *Mobile NixOS* maintainers.

Any suggestions?

**Pull request closed without merging**